src/AzSK.ADO/Framework/Configurations/SVT/ADO/ADO.VariableGroup.json

```diff
@@ -89,6 +89,23 @@
               "MSW"
             ],
             "Enabled": true
+          },
+          {
+            "ControlID": "ADO_VariableGroup_AuthZ_Restrict_Broader_Group_Access_On_VG_With_Secrets",
+            "Description": "Broader groups (contributors, project valid users, etc.) should not have user/administrator privileges on variable group contains secrets.",
+            "Id": "VariableGroup170",
```

Comment:
may be it should be like "...on variable groups containing secrets." or "on variable group which contains secrets."

Reply:
changes done